content/dev/code/notifiers.md

```diff
@@ -35,6 +35,7 @@ Here are some 'quick links' to various sections of this documentation:
 
 5. [Additional Information](#additional-information).  This section has references to additional documentation on the observer/notifier system.
 
+6. [Complete Examples](#complete-examples) Examples of using notifiers and observers.
```

Comment:
```suggestion
6. [Complete Examples](#complete-examples) of using notifiers and observers.
```
```suggestion
6. [Complete Examples](#complete-examples) Examples of using notifiers and observers.
```

Reply:
I dunno why github is showing two suggestions. Maybe just an API glitch.

My point: the word "Examples" is repeated, once inside the link, and again immediately after the link.

Reply:
The items above this are complete sentences after the link.  Changing to make this match prior item.